docs/modules/examples/pages/hotels-app.adoc

```diff
@@ -0,0 +1,133 @@
+
+See the https://docs.datastax.com/en/ragstack/docs/prerequisites.html[Prerequisites] page for more details on finding these values.
+
+== Load the data
+
+. From the root folder, run four Python scripts to populate your database with data collections.
++
+[tabs]
+======
+Python::
++
+[source,bash]
+----
+python -m setup.2-populate-review-vector-collection
+python -m setup.3-populate-hotels-and-cities-collections
+python -m setup.4-create-users-collection
+python -m setup.5-populate-reviews-collection
+----
+
+Result::
++
+[source,console]
+----
+[2-populate-review-vector-collection.py] Finished. 10000 rows written.
+[3-populate-hotels-and-cities-collections.py] Inserted 1433 hotels
+[3-populate-hotels-and-cities-collections.py] Inserted 842 cities
+[5-populate-reviews-collection.py] Inserted 10000 reviews
+----
+======
++
+. Each script populates a different collection in your Astra database, including a collection of precalculated embeddings for vector search.
+
+The application will use these collections to deliver valuable, personalized results to users.
+
+== Run the application
+
+Now that your vector database is populated, run the application frontend to see the results.
+
+. Open a new terminal and start the API server.
++
+[source, bash]
+----
+uvicorn api:app --reload
+----
++
+. Open a new terminal and change directory to the `client` folder.
+Install the node dependencies and start the application.
++
+[source, bash]
+----
+npm install
+npm start
+----
++
+. Open http://localhost:3000 to view the application in your browser.
+Click "Login" in the upper right corner, enter any values for the username and password, and click Login.
++
+. Enter `US` for the country and a US city for the location, and click Search.
++
+. The application lists hotels, including an OpenAI-generated summary of reviews from the reviews collection.
++
+. Selecting "Details" will show more information about the hotel, including a summary based on your Preferences, stored in the users collection.
+
+image::hotels-app.png[Hotels,200,100,align="center"]
+
+[NOTE]
+====
+If your results summaries aren't displaying and you're getting openai errors from pydantic, run `pip install pydantic~=1.10.10` to downgrade pydantic.
+====
+
+== Cleanup
+
+. Use `ctrl+c` to stop the API server and the application.
+. Use the Data API command below or see https://docs.datastax.com/en/astra/astra-db-vector/api-reference/data-api-commands.html#delete-a-collection[delete a collection^] to delete the created collections and make room in your Astra database.
++
+[source,curl]
+----
+curl -v -s --location \
+--request POST "https://ASTRA_DB_ID-ASTRA_DB_REGION.apps.astra.datastax.com/api/json/v1/default_keyspace" \
+--header "X-Cassandra-Token: AstraCS:..." \
+--header "Content-Type: application/json" \
+--header "Accept: application/json" \
+--data '{
+  "deleteCollection": {
+    "name": "hotels"
```

Comment:
Actually the app creates five collections (`hotels`, `cities`, `reviews`, `hotel_reviews_embeddings`, `users`).
A proper cleanup would have the user drop them all.

Maybe better to wrap this in a clealy labeled tiny python script "setup/cleanup_delete_everything.py" or something. (this would be on me in the repo).

Not to mention the _sixth_ collection (LLM cache) that will also the party as soon as the 5-limit is lifted...

Reply:
Good call. 
Here's a PR to add a delete_collection script:
https://github.com/hemidactylus/langchain-astrapy-hotels-app/pull/1